resources/assets/lib/header-v4.tsx

```diff
@@ -53,20 +54,22 @@ export default class HeaderV4 extends React.Component<Props> {
         <div className='header-v4__content'>
           <div className='header-v4__row header-v4__row--title'>
             <div className='header-v4__icon' />
-              <div className='header-v4__title'>
-                <span className='header-v4__title-section'>
-                  {this.props.section}
-                </span>
-                <span className='header-v4__title-action'>
-                  {' '}
-                  {this.props.subSection}
-                </span>
-              </div>
+            <div className='header-v4__title'>
+              <span className='header-v4__title-section'>
+                {this.props.section}
+              </span>
+              <span className='header-v4__title-action'>
+                {' '}
+                {this.props.subSection}
```

Comment:
can use interpolation:
```
` ${this.props.subSection}`
```
(although not sure if better or worse)

Reply:
It'll be gone anyway :dancer: